packages/.eslintrc.json

```diff
@@ -26,7 +27,17 @@
         ],
         "@typescript-eslint/explicit-function-return-type": [
             1,
-            { "allowExpressions": true }
+            {
+                "allowExpressions": true
+            }
         ]
-    }
+    },
+    "overrides": [
+        {
+            "files": ["*.test.ts"],
```

Comment:
Don't add this lint rule to test files where we might choose to test `document.activeElement` on purpose.